src/services/swagger.js

```diff
@@ -0,0 +1,30 @@
+import swaggerJsdoc from 'swagger-jsdoc';
+
+const swaggerSpecs = () => {
+  const options = {
+    definition: {
+      openapi: '3.0.3',
+      swagger: '3.0.3',
+      info: {
+        title: 'ImageUs API',
+        description: 'This is the ImageUs API documentation',
+        license: {
+          name: 'MIT',
+          url: 'https://spdx.org/licenses/MIT.html',
+        },
+      },
+      servers: [
+        {
+          url: 'http://localhost:5000/',
+          description: 'local server',
+        },
```

Comment:
The production server should also be added.

Reply:
I'll add it soon.